pkg/sippyserver/metrics/metrics.go

```diff
@@ -73,33 +96,58 @@ var (
 	}, []string{"component", "network", "arch", "platform"})
 	disruptionVsPrevGAMetric = promauto.NewGaugeVec(prometheus.GaugeOpts{
-		Name: "sippy_disruption_vs_prev_ga",
+		Name: disruptionVsPrevGAMetricName,
 		Help: "Delta of percentiles now vs the 30 days prior to previous release GA date",
-	}, []string{"delta", "release", "compare_release", "platform", "backend", "upgrade_type", "master_nodes_updated", "network", "topology", "architecture"})
+	}, []string{"delta", "release", "compare_release", "platform", "backend", "upgrade_type", "master_nodes_updated", "network", "topology", "architecture", "releaseStatus"})
 	disruptionVsPrevGARelevanceMetric = promauto.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "sippy_disruption_vs_prev_ga_relevance",
 		Help: "Rating of how relevant we feel our data is for regression detection.",
-	}, []string{"release", "compare_release", "platform", "backend", "upgrade_type", "master_nodes_updated", "network", "topology", "architecture"})
+	}, []string{"release", "compare_release", "platform", "backend", "upgrade_type", "master_nodes_updated", "network", "topology", "architecture", "releaseStatus"})
 	disruptionVsTwoWeeksAgo = promauto.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "sippy_disruption_vs_two_weeks_ago",
 		Help: "Delta of percentiles now vs two weeks ago for a given release",
-	}, []string{"delta", "release", "platform", "backend", "upgrade_type", "master_nodes_updated", "network", "topology", "architecture"})
+	}, []string{"delta", "release", "platform", "backend", "upgrade_type", "master_nodes_updated", "network", "topology", "architecture", "releaseStatus"})
 	disruptionVsTwoWeeksAgoRelevanceMetric = promauto.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "sippy_disruption_vs_two_weeks_ago_relevance",
 		Help: "Rating of how relevant we feel our data is for regression detection.",
-	}, []string{"release", "compare_release", "platform", "backend", "upgrade_type", "master_nodes_updated", "network", "topology", "architecture"})
+	}, []string{"release", "compare_release", "platform", "backend", "upgrade_type", "master_nodes_updated", "network", "topology", "architecture", "releaseStatus"})
 )
 
+// getReleases gets all the releases defined in the BQ Releases table if bqc is defined.
+// Otherwise, it falls back to get it from sippy DB
+func getReleases(dbc *db.DB, bqc *bqclient.Client) ([]query.Release, error) {
+	if bqc != nil {
+		releases, err := api.GetReleasesFromBigQuery(bqc)
+		if err != nil {
+			log.WithError(err).Error("error getting releases from bigquery")
+			return releases, err
+		}
+		// Add special release Presubmits for prow jobs
+		releases = append(releases, query.Release{Release: releasePresubmits})
+		return releases, nil
+	}
+	return query.ReleasesFromDB(dbc)
+}
+
+func getReleaseStatus(releases []query.Release, release string) string {
+	releaseStatus := releaseStatusEOL
+	for _, r := range releases {
+		if r.Release == release && len(r.Status) != 0 {
+			releaseStatus = r.Status
```

Comment:
minor and optional suggestion: it would be nice to do a `break` here since you found the release.  And later, you can tell if the release was not found which is probably an erroneous condition.  But if we can guarantee the new `Releases` table will always contain the release, maybe it doesn't matter.

But, if you account for this by defaulting to `releaseStatusEOL` in line 133, if the release is not found, maybe it's fine.